.github/workflows/ci.yml

```diff
@@ -555,6 +555,8 @@ jobs:
         sudo apt install -y --no-install-recommends libxmu-dev libxi-dev libgl-dev libglu1-mesa-dev libgles2-mesa-dev libwayland-dev libxkbcommon-dev libegl1-mesa-dev libosmesa6 mesa-utils libglvnd-dev
     - uses: lukka/get-cmake@latest
     - uses: lukka/run-vcpkg@v10
+      with:
```

Comment:
The checkout a few lines above here says `submodules: true`. Is that still desired?

Reply:
Also for other platforms...